deno.jsonc

```diff
@@ -87,5 +90,18 @@
       "**/wasm_*",
       "**/*.mjs"
     ]
-  }
+  },
+  "workspaces": [
+    "bundle/",
+    "crypto/",
+    "diff/",
+    "logger/",
+    "orm/",
+    "qrcode/",
+    "reactive/",
+    "run/",
+    "testing/",
+    "typing/",
+    "xml/"
```

Comment:
It makes vercel deployments fail with the following:

> error: Workspace member 'bundle/' has no deno.json file ('/vercel/path1/bundle/deno.json')
> Error: Build script failed with exit code 1

I think it's related to https://github.com/denoland/deno/issues/23343 and the fact that the [vercel-community/deno uses 1.42.1 currently](https://github.com/vercel-community/deno/blob/e8d1d546adda2c1a8e0c4c7765e358210168e9d7/src/deno-lambda.ts#L4). 

I know you can pass a `--version` flag to them according to their docs, but I'll prefer it to not do as it does not make sense to break lambdas just because of a conf change which is enterily not related to actual code.

I'll wait for a version bump on their side before merging this

Reply:
vercel-community/deno#155